flask/cli.py

```diff
@@ -369,7 +369,7 @@ def load_app(self):
             app = call_factory(self, self.create_app)
         else:
             if self.app_import_path:
-                path, name = (self.app_import_path.split(':', 1) + [None])[:2]
+                path, name = (re.split(r':(?![\\/])', self.app_import_path) + [None])[:2]
```

Comment:
Should still only split once, `re.split(pattern, value, 1)`.

Reply:
Thank you! I forgot about it. I have revised it according to what you said.